xml/installation-installation-ses_integration.xml

```diff
@@ -55,6 +55,68 @@
     using the command <command>uuidgen</command>.
    </para>
   </important>
+ 
+  <para>
+   For SES deployments that have version 5.5 and above, there is a salt runner
+   that can create all the users and pools and generate a yaml configuration that
+   is needed to integrate with SUSE OpenStack Cloud. The integration runner will
+   create separate users for cinder, cinder backup and glance. Both the cinder
+   and nova services will use the same user, as cinder needs access to create
+   objects that nova uses.
+
+   Usage:
+   To run the SES 5.5 salt runner you need to login as root on the salt admin
+   host.
+
+   salt-run --out=yaml openstack.integrate prefix=mycloud
+
+   The prefix parameter allows you to create pools with that prefix.
+   This allows for multiple cloud deployments to use different users and pools
+   on the same SES deployment.
+  </para>
+  <para>
+   The sample yaml output:
+   <screen>ceph_conf:
+            cluster_network: 10.84.56.0/21
+            fsid: d5d7c7cb-5858-3218-a36f-d028df7b0673
+            mon_host: 10.84.56.8, 10.84.56.9, 10.84.56.7
+            mon_initial_members: ses-osd1, ses-osd2, ses-osd3
+            public_network: 10.84.56.0/21
+            cinder:
+            key: AQCdfIRaxefEMxAAW4zp2My/5HjoST2Y8mJg8A==
+            rbd_store_pool: mycloud-cinder
+            rbd_store_user: cinder
+            cinder-backup:
+            key: AQBb8hdbrY2bNRAAqJC2ZzR5Q4yrionh7V5PkQ==
+            rbd_store_pool: mycloud-backups
+            rbd_store_user: cinder-backup
+            glance:
+            key: AQD9eYRachg1NxAAiT6Hw/xYDA1vwSWLItLpgA==
+            rbd_store_pool: mycloud-glance
+            rbd_store_user: glance
+            nova:
+            rbd_store_pool: mycloud-nova
+            radosgw_urls:
+                - http://10.84.56.7:80/swift/v1 
+                - http://10.84.56.8:80/swift/v1 </screen>
+  <para>
+   Once you have run the openstack.integrate runner, copy the yaml into the
+   ses_config.yml file on the deployer node. Then you need to edit the
+   settings.yml file to enable SUSE Enterprise Storage integration to run and
+   update all of the SUSE OpenStack Cloud service configuration files. The
+   settings.yml resides in ~/openstack/my_cloud/config/ses directory. Open
+   the settings.yml file for editing, uncomment the ses_config_path: parameter,
+   and specify the location on the deployer host containing the ses_config.yml
+   file.
+   <screen>use the example settings.yml as in existing cloud8 docs **
+           use the existing example of commiting settings.yml changes to git **
+           use the existing exmaple of running the Config processor **
+           use the existing exmaple of running ready-deployment **
+           use the existing exmaple of running ardana-reconfigure.yml ** </screen>
+   At this point the rest of the existing SES integration documentation  
+   for running the integration is the same as already documented for Cloud8.</screen> 
```

Comment:
I do not like this wording. maybe we pull out the section with cp,ready,... and just refence some thing like continue on to section x.x.x.  so we could have 1 section that coves new integration, another section existing and they both point to the next steps . Did this make sense?